tests/integration/targets/aci_l3out_logical_interface_profile_eigrp_profile/tasks/main.yml

```
@@ -0,0 +1,188 @@
+# Author: Jason Juenger (@jasonjuenger)
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+# SET VARS
+- name: Set vars
+  ansible.builtin.set_fact:
+    aci_info: &aci_info
+      host: '{{ aci_hostname }}'
+      username: '{{ aci_username }}'
+      password: '{{ aci_password }}'
+      validate_certs: '{{ aci_validate_certs | default(false) }}'
+      use_ssl: '{{ aci_use_ssl | default(true) }}'
+      use_proxy: '{{ aci_use_proxy | default(true) }}'
+      output_level: '{{ aci_output_level | default("info") }}'
+
+# CLEAN ENVIRONMENT
+- name: Remove test tenant before we kickoff
+  cisco.aci.aci_tenant: &tenant_absent
+    <<: *aci_info
+    tenant: ansible_test
+    state: absent
+
+- name: Verify Cloud and Non-Cloud Sites in use.
+  ansible.builtin.include_tasks: ../../../../../../integration/targets/aci_cloud_provider/tasks/main.yml
+
+- name: Execute tasks only for non-cloud sites
+  when: query_cloud.current == []  # This condition will execute only non-cloud sites
+  block:  # block specifies execution of tasks within, based on conditions
+  # SETUP ENVIRONMENT
+  - name: Create domain
+    cisco.aci.aci_domain: &domain_present
+      <<: *aci_info
+      domain: l3outintftest
+      domain_type: l3dom
+      state: present
+
+  - name: Create tenant
+    cisco.aci.aci_tenant: &tenant_present
+      <<: *tenant_absent
+      state: present
+
+  - name: Configure VRF
+    cisco.aci.aci_vrf: &vrf_present
+      <<: *tenant_present
+      vrf: l3outintftest
+
+  - name: Create L3Out
+    cisco.aci.aci_l3out:
+      <<: *vrf_present
+      l3out: l3outintftest
+      domain: l3outintftest
+      route_control: export
+
+  - name: Create node profile
+    cisco.aci.aci_l3out_logical_node_profile: &np_present
+      <<: *tenant_present
+      l3out: l3outintftest
+      node_profile: NODES
+
+  - name: Create interface profile
+    cisco.aci.aci_l3out_logical_interface_profile: &intf_present
+      <<: *np_present
+      interface_profile: INTFS
+
+  - name: Create eigrp interface policy
+    cisco.aci.aci_interface_policy_eigrp: &interface_policy_eigrp_present
+      <<: *tenant_present
+      eigrp_interface: ansible_eigrp_interface
+      description: eigrp interface policy for ansible_tenant tenant
+      bandwidth: 100000
+      control_state: [bfd, nexthop_self, passive, split_horizon]
+      delay: 10
+      delay_unit: picoseconds
+      hello_interval: 10
+      hold_interval: 30
+      state: present
+
+  # BEGIN WITH TESTS (ADD EIGRP profile TO INTERFACE PROFILE)
+  - name: Add eigrp profile to interface profile (check mode)
+    cisco.aci.aci_l3out_logical_interface_profile_eigrp_profile: &eigrpIfP_present
```

can we use consistent snake case
```suggestion
    cisco.aci.aci_l3out_logical_interface_profile_eigrp_profile: &eigrp_interface_profile_present
```

---

this comments applies to entire test file

---

fixed

---

I do not see these changes